docker/service/mongod/run

```diff
@@ -0,0 +1,2 @@
+#!/bin/sh
+exec /usr/bin/mongod
```

Comment:
Should the application container be running mongodb? Normally I would run mongo in a separate container/standalone service.

Having it embedded in the application container makes it harder to have persistent data.

Reply:
Persistent storage is already possible, in two specific ways:

* One can update the `config/config.default.php` file with the correct mongo address. This keeps the in-container mongo instance running, but does not use it.
* One can run with the in-container mongo instance, but use a docker volume for that: `docker run -v $(pwd)/mongo-data:/data/db`).

Would it suffice if I add a README.md file in the `docker/` folder specifying these options?

Reply:
Yes you could do those things, but you will not be able to restart or manage the in-container mongo at all though, and when not in use it will burn a non-zero amount of resources.

Is there a reason to not use `docker-compose`? It sets the user up for success better in the future.